pkg/cover/html.go

```diff
@@ -999,7 +1022,7 @@ var coverTemplate = template.Must(template.New("").Parse(`
 		<li><span class="hover">
 			{{if $file.Covered}}
 				<a href="#{{$file.Path}}" id="path/{{$file.Path}}" onclick="onFileClick({{$file.Index}})">
-					{{$file.Name}}
+					{{$file.Name}}[{{$file.Module}}]
```

Comment:
Is `Module` always non-empty? If not, maybe only print brackets if it's really a module?